sdk/common/src/main/java/com/mesosphere/sdk/testutils/TestConstants.java

```diff
@@ -40,6 +40,7 @@
     public static final String HAS_VIP_LABEL = "hasvip";
     public static final String ROOT_DISK_TYPE = "ROOT";
     public static final String MOUNT_DISK_TYPE = "MOUNT";
+    public static final String MOUNT_SOURCE_ROOT = "/mnt/src";
```

Comment:
looks like this is only used in `ResourceBuilderTest`, may as well just keep it there